docs/content/posts/2025-05-28-release-kubernetes-operator-1.12.0.md

```diff
@@ -0,0 +1,61 @@
+---
+title:  "Apache Flink Kubernetes Operator 1.12.0 Release Announcement"
+date: "2025-05-28T18:00:00.000Z"
+authors:
+- gyfora:
+  name: "Gabor Somogyi"
+aliases:
+- /news/2025/05/28/release-kubernetes-operator-1.12.0.html
+---
+
+The Apache Flink community is excited to announce the release of Flink Kubernetes Operator 1.12.0!
+The version brings a number of important fixes and improvements to both core and autoscaler modules.
+
+We encourage you to [download the release](https://flink.apache.org/downloads.html) and share your experience with the
+community through the Flink [mailing lists](https://flink.apache.org/community.html#mailing-lists) or
+[JIRA](https://issues.apache.org/jira/browse/flink)! We're looking forward to your feedback!
+
+## Highlights
+
+### Enhanced Error Visibility and Event Reporting
+
+- **Improved Error Transparency**: The operator now exposes detailed error messages for deployment issues, aiding in faster diagnosis and resolution.
+- **Kubernetes Event Integration**: Failures such as job transitions to the FAILED state and job restarts now generate Kubernetes events, providing better observability and alerting capabilities.
+
+### Flink Version Compatibility and Configuration Enhancements
+
+- **Flink 1.19 `config.yaml` Support**: The operator adds compatibility with Flink 1.19's `config.yaml` format, ensuring seamless integration with the latest Flink configurations.
+- **Upfront YAML Configuration Validation**: Introduced validation for Flink 2.x configurations, enabling early detection of misconfigurations before deployment.
+
+### Deployment and Upgrade Process Improvements
+
+- **Init and Sidecar Container Configuration**: Users can now configure init and sidecar containers within their deployments, offering greater flexibility in customizing the Flink runtime environment.
+- **Observer Optimization**: The observer component has been refined to avoid checking all JobManager replicas, reducing unnecessary overhead and improving performance.
```

Comment:
I would combine these two in `Helm Chart Improvements` these are released mostly to the operator itself and not the Flink runtime/jobs/deployments.

Also We can just say `config.yaml` support for the operator configuration (no need to mention 1.19) we can reference to at as standard yaml format.